src/main/java/com/airdnb/clone/domain/stay/StayRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.airdnb.clone.domain.stay;
+
+import com.airdnb.clone.domain.stay.entity.Stay;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface StayRepository extends JpaRepository<Stay, Long> {
+}
```

Comment:
```suggestion
@Repository
public interface StayRepository extends JpaRepository<Stay, Long> {
}
```

여기도 명확성을 위해 추가해보는게 어떨까요